sklearn/ensemble/weight_boosting.py

```diff
@@ -353,6 +353,24 @@ class AdaBoostClassifier(BaseWeightBoosting, ClassifierMixin):
     feature_importances_ : array of shape = [n_features]
         The feature importances if supported by the ``base_estimator``.
 
+    Examples
+    --------
+    >>> from sklearn.ensemble import AdaBoostClassifier
+    >>> from sklearn.datasets import make_classification
+    >>> X, y = make_classification(n_samples=1000, n_features=4,
+    ...                            n_informative=2, n_redundant=0,
+    ...                            random_state=0, shuffle=False)
+    >>> clf = AdaBoostClassifier(n_estimators=100, random_state=0)
+    >>> clf.fit(X, y)  # doctest: +NORMALIZE_WHITESPACE
+    AdaBoostClassifier(algorithm='SAMME.R', base_estimator=None,
+            learning_rate=1.0, n_estimators=100, random_state=0)
+    >>> clf.feature_importances_
+    array([0.28 0.42 0.14 0.16])
+    >>> clf.predict([[0, 0, 0, 0]])
+    array([1])
+    >>> clf.score(X, y)
```

Comment:
try to use ``# doctest: +ELLIPSIS`` when possible (to avoid failures on other platforms)

Reply:
@qinhanmin in which line are you referring to? 367, 369 and 371?

Reply:
I think you can always use it when you're trying to output some floats :)

Reply:
I had the impression that you add it when then expected floating points are bigger that 4 and you use the three dots

Reply:
> I had the impression that you add it when then expected floating points are bigger that 4 and you use the three dots

I don't think there's a clear standard in scikit-learn. The point here is to avoid test failures.
Maybe I'll add it in L367, L371, L934, L936, L938

Reply:
So let's doit